source/MRViewer/MRSurfaceContoursWidget.cpp

```diff
@@ -380,24 +340,14 @@ bool SurfaceContoursWidget::appendPoint( const std::shared_ptr<VisualObject>& ob
 
     auto onAddPointAction = [this, &obj, &triPoint] ()
     {
-        auto& contour = pickedPoints_[obj];
-
+        auto actionPtr = AddRemovePointHistoryAction::append( *this, obj, triPoint );
```

Comment:
It's not clear that the action is actually performed here, one could assume it's only a history action being constructed.

Reply:
renamed